### ISSUE TITLE
Survey Form Challenge: Script Bug

### ISSUE THREAD

**Issue by truonglybach** [freeCodeCamp/testable-projects-fcc#505]
#### Issue Description

Although the testing script is saying that I'm missing the id="dropdown" requirement and the name attribute for my radio group for the survey form challenge, my code fulfills all the requirements.

#### Browser Information

* Browser Name, Version: Google Chrome, Version 67.0.3396.99
* Operating System: Windows 7 Home Premium
* Mobile, Desktop, or Tablet: Desktop

#### Your Code / Link to Your Pen

```
 https://codepen.io/truonglybach/pen/wxqGoe?editors=1100

```

#### Screenshot

![surveyformfcc](https://user-images.githubusercontent.com/39682113/43618271-1e1fcfbe-967c-11e8-849f-f8f29227d83e.png)

**Comment by tbushman** [comment]
@truonglybach Having several forms on one page can be tricky. To simplify this project, you can use one form, making sure to wrap all form elements inside of it.

**Comment by truonglybach** [comment]
I did what you advised, but the score had actually dropped! I checked for all 16 user stories that were stated on the assignment (https://learn.freecodecamp.org/responsive-web-design/responsive-web-design-projects/build-a-survey-form) and I fulfilled every requirement.
I also noticed that the user-story-checking script checks for 17 items instead of 16 and was wondering if my problem was related to that.
<img width="1101" alt="screen shot 2018-08-06 at 12 42 04 am" src="https://user-images.githubusercontent.com/39682113/43703447-a8fac5a2-9911-11e8-9e77-671af93642ee.png">
<img width="604" alt="screen shot 2018-08-06 at 12 42 25 am" src="https://user-images.githubusercontent.com/39682113/43703453-ab6a8e30-9911-11e8-9e23-fc74363982ca.png">
